### PR TITLE
New design for the site manager

### PR DESCRIPTION
**Pull request by mnapoli**
**Do not merge**

This is an alternative design for #8001 (redesign of the site manager).

I've taken a different approach here to stay more in line with the original mockup but less borders (inspired a bit by "cards" of material design: the outline of the block is done by the shadow).

Feel free to :-1: or suggest alternative routes, I'm just trying out things ;)

capture d ecran 2015-05-26 a 14 28 28

### CODE REVIEW

**Comment by diosmosis** [comment]
Looks cool, :+1:

**Comment by tsteur** [comment]
Prefer this one as it scales more than #8001 . We cannot add an unlimited number of columns in #8001 whereas this one here scales. Plugins / Types can add more information to the 2 existing columns in the middle making it much more usable and look better. Also it allows us to show different information eg App-ID for mobile apps and URL's for websites. In #8001 we wouldn't be able to show any of those information. The layout of #8001 gets way too wide making it look weird and a bit unprofessional

**Comment by tsteur** [comment]
@mnapoli is there much missing or could we merge?

**Comment by mnapoli** [comment]
@tsteur FYI I think it should be responsive out of the box (used bootstrap grid) but the admin screens have a fixed width so it cannot be really used right now.

This PR can't be merged yet because it is based on #7960 (forms redesign). I need to adjust a few things there (comments) + check for regressions. I'll focus on this to try to merge this ASAP.

File: plugins/PrivacyManager/templates/privacySettings.twig [hunk 1]
```diff
@@ -7,266 +7,276 @@
         help-url="http://piwik.org/docs/privacy/">{{ 'PrivacyManager_TeaserHeadline'|translate }}</h2>
     <p>{{ 'PrivacyManager_Teaser'|translate('<a href="#anonymizeIPAnchor">',"</a>",'<a href="#deleteLogsAnchor">',"</a>",'<a href="#optOutAnchor">',"</a>")|raw }}
         {{'PrivacyManager_SeeAlsoOurOfficialGuidePrivacy'|translate('<a href="http://piwik.org/privacy/" rel="noreferrer"  target="_blank">','</a>')|raw }}</p>
+
     <h2 id="anonymizeIPAnchor">{{ 'PrivacyManager_UseAnonymizeIp'|translate }}</h2>
     <form method="post" action="{{ {'action':'saveSettings', 'form':'formMaskLength', 'token_auth':token_auth} | urlRewriteWithParameters }}" id="formMaskLength">
```
Comment (tsteur) [inline plugins/PrivacyManager/templates/privacySettings.twig]: doesn't this mean the token_auth would be sent as a GET parameter? It should be sent as POST to not appear in logs etc

File: plugins/PrivacyManager/templates/privacySettings.twig [hunk 2]
```diff
@@ -7,266 +7,276 @@
...
+            </div>
+            <label class="radio">
+                <input id="anonymizeIPEnable-1" type="radio" name="anonymizeIPEnable" value="1" {% if anonymizeIP.enabled == '1' %}checked {% endif %}/>
+                {{ 'General_Yes'|translate }}
+            </label>
+            <label class="radio">
+                <input class="indented-radio-button" id="anonymizeIPEnable-0" type="radio" name="anonymizeIPEnable" value="0" {% if anonymizeIP.enabled == '0' %} checked {% endif %}/>
+                {{ 'General_No'|translate }}
+            </label>
+            <input type="hidden" name="token_auth" value="{{ token_auth }}"/>
```
Comment (tsteur) [inline plugins/PrivacyManager/templates/privacySettings.twig]: Token is also set again here and in the form? Ideally we would maybe replace it by an `nonce` but that's not part of redesign :)

File: plugins/PrivacyManager/templates/privacySettings.twig [hunk 3]
```diff
@@ -7,266 +7,276 @@
...
+            <input role="yes" type="button" value="{{ 'General_Ok'|translate }}"/>
+        </div>
+        <div class="ui-confirm" id="confirmPurgeNow">
+            <h2>{{ 'PrivacyManager_PurgeNowConfirm'|translate }}</h2>
+            <input role="yes" type="button" value="{{ 'General_Yes'|translate }}"/>
+            <input role="no" type="button" value="{{ 'General_No'|translate }}"/>
+        </div>
+        <h2 id="deleteLogsAnchor">{{ 'PrivacyManager_DeleteDataSettings'|translate }}</h2>
+        <p>{{ 'PrivacyManager_DeleteDataDescription'|translate }} {{ 'PrivacyManager_DeleteDataDescription2'|translate }}</p>
+        <form method="post" action="{{ {'action':'saveSettings','form':'formDeleteSettings','token_auth':token_auth} | urlRewriteWithParameters }}" id="formDeleteSettings">
```
Comment (tsteur) [inline plugins/PrivacyManager/templates/privacySettings.twig]: `token_auth` set here in action as well which causes it to be sent as GET parameter?

**Comment by tsteur** [comment]
Didn't have a detailed look re HTML or so apart from security. Few comments re `token_auth` otherwise looks good to me.

**Comment by mnapoli** [comment]
@tsteur could you open a separate issue for security stuff, this is code that I haven't changed.

I have tested the forms, I think this is ready to be merged.

**Comment by tsteur** [comment]
sweet, just noticed it was in #7960 anyway and not in this PR. Looked like it was added but was there already. Is the *do not merge* still valid?

**Comment by mnapoli** [comment]
yeah the diff on github is really not as good as the real git diff, it shows changes on big blocks where it's only one e.g. one line in the block that has changed.

**Comment by mattab** [comment]
:+1:  nice PR & UI improvement!

**Pull request merged**